### PR TITLE
1.10.3 update dependencies

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

I've attempted to update the dependencies, following changes in `setup.py` seen here:
https://github.com/apache/airflow/compare/1.10.1...apache:1.10.3#diff-2eeaed663bd0d25b7e608891384b7298

closes #25

### CODE REVIEW

**Comment by sodre** [comment]
@xylar, please feel free to add yourself as a comaintainer once you're done since I am not using airflow at the moment.

Once the build is ready, ping me and I'll merge.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,38 +29,41 @@ outputs:
...
+        - requests >=2.20.0,<3
         - setproctitle >=1.1.8,<2
-        - sqlalchemy >=1.1.15,<1.2.0
-        - tabulate >=0.7.5,<0.8.0
-        - tenacity ==4.8.0
+        - sqlalchemy >=1.1.15,<1.3.0
+        - tabulate 0.7.5,<0.9
+        - tenacity ==4.12.0
+        - text-unidecode ==1.2
+        - typing  # [py<35]
```
Comment (xylar) [inline recipe/meta.yaml]: @sondre, this is following the original `setup.py`
https://github.com/apache/airflow/blob/1.10.3/setup.py#L325

If you think it's also needed for python 3.7, let me know.

**Comment by xylar** [comment]
@sodre, I've added myself.  It looks like you're right and some packages upstream need some loving, too. I'll see what I can do.

**Comment by sodre** [comment]
Closed by https://github.com/conda-forge/airflow-feedstock/commit/2cf8845d8854c766ff9b20aed14b538f31c7864a

**Comment by xylar** [comment]
I think we still want to do this one.  I'll work on it today.

**Comment by xylar** [comment]
Support for python 3.7 is still not possible because these library versions are not available:
* lxml>=3.6.0,<4.0
* flask-swagger==0.2.13
* tenacity==4.12.0

It seems like it may be possible to build this version of tenacity for py37 (unlike 4.8.0, see #29).  I will look into the other 2.

**Comment by xylar** [comment]
Working on these here:
https://github.com/conda-forge/tenacity-feedstock/pull/17
and here:
https://github.com/conda-forge/flask-swagger-feedstock/pull/3

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (sodre) [reply recipe/meta.yaml]: @xylar, I am okay not building for python 3.7. 

Feel free to merge.

Comment (xylar) [reply recipe/meta.yaml]: Okay, I'll go ahead and merge.  We could always do another build that supports python 3.7 once the other two libraries are in place.

**Pull request merged**